{{ cookiecutter.project_slug }}/api/{{ cookiecutter.project_slug }}/urls.py

```diff
@@ -1,24 +1,25 @@
 from django.conf import settings
 from django.contrib import admin
+from django.contrib.staticfiles.urls import static
 from django.urls import include, path, re_path
 
 
 PLATFORM_PREFIX = "_platform"
 API_PREFIX = "api"
 DOCS_PREFIX = "docs"
 
-admin_urlpatterns = [
-    path("admin/", admin.site.urls),
-]
-
 api_v1_urlpatterns = [
     path(
         f"{API_PREFIX}/v1/accounts/",
         include(("{{ cookiecutter.project_slug }}.apps.accounts.api.v1.urls", "accounts"), namespace="api-v1-accounts"),
     )
 ]
 
-urlpatterns = admin_urlpatterns + api_v1_urlpatterns
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    *api_v1_urlpatterns,
+    *static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT),
```

Comment:
Two questions:
1) Do we need to serve media with Django in production (when DEBUG is off)?
2) I don't see an URL for static files (STATIC_URL). You've deleted it in [line 66](https://github.com/django-stars/backend-skeleton/pull/67/files#diff-967b13624d90124c8ff7cc85dde2191f3278db85cc6538670a42abe978cd1cd7L66) but didn't add it.

Reply:
1 static function has DEBUG mode check inside [def static(prefix, view=serve, **kwargs):. ](https://github.com/django/django/blob/main/django/conf/urls/static.py#L10)
2 This configuration serves static files as expected without serving  STATIC_URL explicitly.